.travis.yml

```diff
@@ -216,11 +216,12 @@ matrix:
           - /root/travis_bootstrap
           - /hab/studios/home--travis--build--habitat-sh
           - /root/.cargo
-      before_install:
+      script:
         - ./support/ci/fast_pass.sh || exit 0
+      before_deploy:
         - if [[ ! -x ./support/ci/deploy.sh ]]; then chmod +x ./support/ci/deploy.sh; fi
         - openssl aes-256-cbc -K $encrypted_50e90ce07941_key -iv $encrypted_50e90ce07941_iv -in ./support/ci/habitat-srv-admin.enc -out /tmp/habitat-srv-admin -d
-      script:
+      deploy:
         - sudo ./support/ci/deploy.sh
```

Comment:
I believe this needs to be:
```
deploy:
  provider: script
  script: sudo ./support/ci/deploy.sh 
```
I'm not certain this will fix PRs though because the `openssl` command will still run for non master PRs if `fast_pass` succeeds. You can test this by marking this PR `DO NOT MERGE` and including a small change to the supervisor (even adding a comment). That should get this past `fast_pass` and validate this in a PR.